src/Microsoft.Graph/Generated/callrecords/requests/CallRecordRequest.cs

```diff
@@ -39,45 +39,50 @@ public partial class CallRecordRequest : Microsoft.Graph.BaseRequest, ICallRecor
         /// </summary>
         /// <param name="callRecordToCreate">The CallRecord to create.</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> for the request.</param>
         /// <returns>The created CallRecord.</returns>
-        public System.Threading.Tasks.Task<CallRecord> CreateAsync(CallRecord callRecordToCreate)
+        public async System.Threading.Tasks.Task<CallRecord> CreateAsync(CallRecord callRecordToCreate, CancellationToken cancellationToken = default(CancellationToken))
         {
-            return this.CreateAsync(callRecordToCreate, CancellationToken.None);
+            this.ContentType = Constants.ContentTypes.JsonContentType;
+            this.Method = "POST";
+            var newEntity = await this.SendAsync<CallRecord>(callRecordToCreate, cancellationToken).ConfigureAwait(false);
+            this.InitializeCollectionProperties(newEntity);
+            return newEntity;
         }
 
         /// <summary>
-        /// Creates the specified CallRecord using POST.
+        /// Creates the specified CallRecord using POST and returns a <see cref="GraphResponse{CallRecord}"/> object.
         /// </summary>
         /// <param name="callRecordToCreate">The CallRecord to create.</param>
         /// <param name="cancellationToken">The <see cref="CancellationToken"/> for the request.</param>
-        /// <returns>The created CallRecord.</returns>
-        public async System.Threading.Tasks.Task<CallRecord> CreateAsync(CallRecord callRecordToCreate, CancellationToken cancellationToken)
+        /// <returns>The <see cref="GraphResponse{CallRecord}"/> object of the request.</returns>
+        public async System.Threading.Tasks.Task<GraphResponse<CallRecord>> CreateResponseAsync(CallRecord callRecordToCreate, CancellationToken cancellationToken = default(CancellationToken))
         {
-            this.ContentType = "application/json";
+            this.ContentType = Constants.ContentTypes.JsonContentType;
             this.Method = "POST";
-            var newEntity = await this.SendAsync<CallRecord>(callRecordToCreate, cancellationToken).ConfigureAwait(false);
-            this.InitializeCollectionProperties(newEntity);
-            return newEntity;
+            return await this.SendAsyncWithGraphResponse<CallRecord>(callRecordToCreate, cancellationToken).ConfigureAwait(false);
         }
 
         /// <summary>
         /// Deletes the specified CallRecord.
         /// </summary>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> for the request.</param>
         /// <returns>The task to await.</returns>
-        public System.Threading.Tasks.Task DeleteAsync()
+        public async System.Threading.Tasks.Task DeleteAsync(CancellationToken cancellationToken = default(CancellationToken))
         {
-            return this.DeleteAsync(CancellationToken.None);
+            this.Method = "DELETE";
```

Comment:
the methods excepted being transformed to constants? 🤣

Reply:
Resolved via https://github.com/microsoftgraph/msgraph-sdk-dotnet/pull/905/commits/15dba2f53329af2e682ac87ff9dc229d89b48fc2